.github/workflows/ironic-image-build-pr.yml

```
@@ -0,0 +1,43 @@
+name: Build Ironic Image on PRs
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize, ready_for_review]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        centos_version:
+          - stream9
+          - stream10
```

```suggestion
        - stream9
        - stream10
```
k8s identation in all the lists, also in  `steps:`